scripts/play.php

```diff
@@ -400,14 +400,14 @@ function changeDetection(filename,copylink=false) {
 <div class="play">
 <?php if($view == "byspecies" || $view == "date") { ?>
 <div style="width: auto;
-   text-align: center">
+   text-align: center"> 
    <form action="views.php" method="GET">
       <input type="hidden" name="view" value="Recordings">
       <input type="hidden" name="<?php echo $view; ?>" value="<?php echo $_GET['date']; ?>">
-      <button <?php if(!isset($_GET['sort']) || $_GET['sort'] == "alphabetical"){ echo "style='background:#9fe29b !important;'"; }?> class="sortbutton" type="submit" name="sort" value="alphabetical">
+      <button <?php if(!isset($_GET['sort']) || $_GET['sort'] == "alphabetical"){ echo "class='sortbutton active !important;'"; }?> type="submit" name="sort" value="alphabetical">
          <img src="images/sort_abc.svg" title="Sort by alphabetical" alt="Sort by alphabetical">
       </button>
-      <button <?php if(isset($_GET['sort']) && $_GET['sort'] == "occurrences"){ echo "style='background:#9fe29b !important;'"; }?> class="sortbutton" type="submit" name="sort" value="occurrences">
+      <button <?php if(isset($_GET['sort']) && $_GET['sort'] == "occurrences"){ echo "class='sortbutton active !important;'"; }?> type="submit" name="sort" value="occurrences">

```

Comment:
By declaring the Class='sortbutton' here (407 & 410) I get a weird cutoff from the sort pictures when I hover over them.
If you declare 'sortbutton' in the DIV on line 402, this gets fixed.
Class='Active' should indeed be declared on 407 & 410. 
![image](https://github.com/Nachtzuster/BirdNET-Pi/assets/145980686/4cb54b1d-9698-40be-a96b-b8ed0af26287)

Reply:
Sorry mistake from my part when copying your code I'll correct

Reply:
The issue if I declare in line 402 is that I get a big non esthetic bar ; i'll push it